.github/ISSUE_TEMPLATE/unestimated-sub-task.md

```diff
@@ -2,7 +2,7 @@
 name: 🧩  Unestimated sub-task
 about: "Specify an unestimated sub-task. (Avoid comments.  Use only as prescribed.)"
 title: ''
-labels: 'sub-task,:release,#mdm'
+labels: '~sub-task'
```

Comment:
To be consistent, we could say we don't add special characters to `story`, `bug`, or `sub-task` because they are mutually exclusive. No issue should ever have one of these labels. By excluding the special characters, we are making clear it's not intended to be used as a standard noun/adjective/verb label. Does that hold water?

Reply:
I would say i agree that `#g-mdm,#g-cx` have a purpose that indicates which team board it's on and should never overlap, I agree `:product,:release` for the same idea where they are mutually exclusive indicating it's on product or eng responsibility in the lifecycle. For all other labels I think these prefixes are not helping. I have a ~mac and ~windows ticket the product board right now. There are some tickets they could have both. Some Tickets could both be frontend and backend simultaneously and I think for anything outside of designating a board we should get rid of all the extra prefixes imo.

Reply:
Thanks for your feedback. If you put in a PR to the handbook with that change, we can move the conversation there and include other folks. I personally like the prefixes, but I'm interested to hear what others think and always open to revising. 

For now, let's leave this as-is since it's not the topic of this PR.